src/plugins/image/imagePlugin.ts

```diff
@@ -95,7 +103,13 @@ export class ImagePlugin extends TemplatePlugin {
         return `
             <pic:pic xmlns:pic="http://schemas.openxmlformats.org/drawingml/2006/picture">
                 <pic:nvPicPr>
-                    <pic:cNvPr id="0" name="${name}"/>
+                    <pic:cNvPr id="${imageId}" name="${name}" ${descrTag}>
```

Comment:
Are you sure this part is required? I don't see it when I add a picture on my machine.

Reply:
I created a word document and extracted the ZIP, and got the below.
Microsoft Office 365 - Word Version 2302 (build 16130.20332)
```
<wp:docPr id="1" name="Image 1" descr="This is AltText"/>
<wp:cNvGraphicFramePr>
<a:graphicFrameLocks xmlns:a="http://schemas.openxmlformats.org/drawingml/2006/main" noChangeAspect="1"/>
</wp:cNvGraphicFramePr>
<a:graphic xmlns:a="http://schemas.openxmlformats.org/drawingml/2006/main">
<a:graphicData uri="http://schemas.openxmlformats.org/drawingml/2006/picture">
<pic:pic xmlns:pic="http://schemas.openxmlformats.org/drawingml/2006/picture">
<pic:nvPicPr>
<pic:cNvPr id="1" name="Image 1" descr="This is AltText"/>
<pic:cNvPicPr/>
</pic:nvPicPr>
```